app/Importer/AssetImporter.php

```diff
@@ -121,7 +121,7 @@ public function createAssetIfNotExists(array $row)
 
             // If we have a target to checkout to, lets do so.
             if(isset($target)) {
-                $asset->fresh()->checkOut($target);
+                $asset->fresh()->checkOut($target, $this->user_id);
```

Comment:
This is going to sound *SUPER* corny - but, can we get a comment here explaining that `$this->user_id` is actually set by the `ItemImporter` (or is it `Importer`?) class? Just that's not visibly clear when you look at it.